astropy/tests/plugins/display.py

```diff
@@ -0,0 +1,142 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""
+This plugin provides customization of the header displayed by pytest for
+reporting purposes.
+"""
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+import os
+import sys
+import datetime
+import locale
+import math
+from collections import OrderedDict
+
+from ...extern import six
+
+from ..helper import ignore_warnings
+from ...utils.introspection import resolve_name
+
+
+PYTEST_HEADER_MODULES = OrderedDict([('Numpy', 'numpy'),
+                                     ('Scipy', 'scipy'),
+                                     ('Matplotlib', 'matplotlib'),
+                                     ('h5py', 'h5py'),
```

Comment:
please do move these to the astropy specific conftest.py, so neither h5py, nor pandas are inherited by affiliated packages. Actually maybe even scipy and mpl can be moved (see https://github.com/astropy/pyregion/pull/120#issuecomment-336489498)

Reply:
I plan to open a separate issue/PR for this once this one is merged, if that's okay.